static/js/network.js

```diff
@@ -80,6 +90,27 @@ var nodeIds, shadowState, nodesArray, nodes, edgesArray, edges, network;
         addNode(data.id, data.group, data.label, 200, -200);
     }
 
+    function activateCurrentNode(){
```

Comment:
can we select multiple nodes? if so, only the latest selected is activated?
selected nodes seems like it's an array, so if the gui allows multiple selection, that seems like it might cause confusion.